apps/correct-mistakes/fields/about-error.js

```diff
@@ -80,96 +68,80 @@ module.exports = {
       value: 'fields.gender-error.legend'
     },
     options: [{
-      value: 'female',
-      label: 'fields.gender-error.options.female.label'
+      value: 'female'
```

Comment:
This works for options too? (You would *hope* it would.)
Is this tested anywhere?